RELEASE-NOTES.md

```diff
@@ -9,6 +9,11 @@ The project release numbers follow [Semantic Versioning](http://semver.org/spec/
 
 ## [Unreleased] - Release date yyyy-mm-dd
 
+## Added
+- Added variable to allow removing of implicit link directories added to CUDA executables
```

Comment:
Reword suggestion:

>  - Added variable `BLT_CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES_EXCLUDE` to allow removing 
implicit link directories added to CUDA executables by CMake. See the following example host-config:
  `host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_nvcc_c++17.cmake`.